recipe/meta.yaml

```diff
@@ -45,8 +45,8 @@ about:
   description: |
     This library provides an iconv() implementation, for use on systems which don't have one,
     or whose implementation cannot convert from/to Unicode.
-  doc_url: https://www.gnu.org/savannah-checkouts/gnu/libiconv/documentation/libiconv-1.15/iconv.1.html
-  dev_url: http://git.savannah.gnu.org/cgit/libiconv.git/tree/
+  doc_url: https://www.gnu.org/savannah-checkouts/gnu/libiconv/documentation/libiconv-{{ version }}/iconv.1.html
+  dev_url: http://git.savannah.gnu.org/cgit/libiconv.git/tag/?h=v{{ version }}
```

Comment:
I _think_ we want `dev_url` to point to main/master (in this case tree)?

Reply:
Updated dev_url and license